tests/integration/integration_steps/fabric_steps.py

```diff
@@ -105,3 +111,27 @@ def assert_data_model_in_fabric():
 def assert_data_model_update():
     # Assert the data model changes including versions and last updated timestamps are propagated to a Fabric lakehouse
     pass
+
+
+def assert_events_data_in_fabric(
+    events_path: str,
+    events_dataframe: pd.DataFrame,
+    azure_credential: DefaultAzureCredential,
+):
+    # Assert events data is populated in a Fabric lakehouse
+    events_from_lakehouse = read_deltalake_timeseries(events_path, azure_credential)
```

Comment:
I remember you bringing this up did we want to change the name of this method now or is that expected in another PR?

Reply:
Let's leave that to another PR, we can make some changes to consolidate the fabric methods but that shouldn't be in this PR